### ISSUE TITLE
Couldn't download pinterest album [ console : Cannot read property 'replace' of null ]

### ISSUE THREAD

**Issue by pavanpwm** [inDream/DownAlbum#27]
fetcher.js/Line 1700  :  g.resource = type.replace(/Feed|Page/g, '') + 'FeedResource'; 
// Uncaught TypeError: Cannot read property 'replace' of null

Please help!!

**Comment by Unicom3000** [comment]
same

**Comment by Bazjoe** [comment]
Same here. Unable to download album

**Comment by timbuktu234** [comment]
DownAlbum stopped working on Pinterest around December 23, 2020

**Comment by tracymc** [comment]
@inDream Yep, I'm having the same issue. Has anyone tried other browsers with scripts? I tried FF and the dialog box opened but nothing actually happened.

**Comment by pavanpwm** [comment]
https://www.wfdownloader.xyz/

Use this instead guys

**Comment by malenked** [comment]
Thanks pavanpwn although I had to find this [pinterest downloader](https://www.wfdownloader.xyz/blog/how-to-bulk-download-images-and-videos-from-pinterest) tutorial showing how to use it.
